src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_utils.py

```diff
@@ -229,7 +229,7 @@ def test_load_cert_file(self):
         try:
             blob, thumbprint = load_cert_file(pfx_file, testpassword)
         except CLIInternalError as e:
-            self.assertTrue(e.error_msg.error_msg.__contains__('Invalid password or PKCS12 data'))
+            self.assertTrue(e.error_msg.error_msg in 'Invalid password or PKCS12 data')
```

Comment:
You could use [`assertIn`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn)